app/main.html

```diff
@@ -21,6 +21,6 @@
     <div role="app"></div>
 
     <script type="application/javascript" src="node_modules/react-mdl/extra/material.min.js"></script>
-    <script type="application/javascript" src="main.js"></script>
+    <script type="application/javascript" src="app.js"></script>
```

Comment:
Sorry, I forgot to integrate it with main.js. I didn't want to change main.js directly. I'll make the changes.